README.md

```diff
@@ -13,3 +13,19 @@ functionality.
 
 - [inject-go-binary](inject-go-binary/): Demos adding building and injecting a Go binary + systemd unit
 - [tailscale](tailscale/): Demos https://tailscale.com/download/linux/fedora
+
+## Running an example
+
+- Build an image using an example from this repo and push it to an image registry:
+  ```
+  set IMAGE (podman build $EXAMPLE -q)
+  podman push $IMAGE quay.io/$USER/$EXAMPLE
+  ```
+
+- Setup a system that has `rpm-ostree` installed. One possibility is [using `virt-install`](https://docs.fedoraproject.org/en-US/fedora-coreos/getting-started/#_booting_on_a_local_hypervisor_libvirt_example).
```

Comment:
There's also bits of this here
https://coreos.github.io/rpm-ostree/layering/#download-fedora-coreos-next-stream

Hmmm...perhaps what we should really do is try to centralize this in the official FCOS documentation.

Reply:
Let's ignore this one for now